src/core/ddsi/src/ddsi_config.c

```diff
@@ -695,6 +695,8 @@ static int if_network_interfaces(struct cfgst *cfgst, void *parent, struct cfgel
   new->cfg.automatic = false;
   new->cfg.name = NULL;
   new->cfg.address = NULL;
+  new->cfg.presence_required = true;
```

Comment:
The values to which the fields are initialised — and so for most fields, whether they are initialised at all — here are irrelevant because all the missing values are set from the default value specified in configuration table.

Strings do need to be initialised to a null pointer because it'll try to free them in the configuration parsing error handling paths.

Doesn't make setting the values here incorrect, but I thought it worth pointing out.